### ISSUE TITLE
[spell] All casting spells goes speedy speedy (BB #1197)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1197]
This issue was migrated from bitbucket.
**Original Reporter:** Dikkedeur
**Original Date:** 31.12.2015 20:11:02 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** invalid
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197

<hr>

how to make a proper title of a bug that is strange.. well you cant :+1: 

so what is the bug...  all spells that have a cast time dont get proper cast time.
With that i mean the following.

You take for example a mage, learn him spells, dont set ANY talent points, just to be sure.
Now, go to a mob, and cast a spell on it.

you see the animation bar, most of the time after roughly 60% you see the casting bar go BOOM to full casting.
So in short, it doesnt take for example 3,5 seconds to fire a fireball rank10 to a mob, but about 1,5 seconds to 1,8 seconds.

now take rank 1 fireball, it has a casting time of 1,5 seconds, its almost good but still not 1,5.
tested it on a new lvl 1 mage.
its roughly 0.75% of a seconds.

so in short it looks like all casting has a speed of 2x what tooltip is telling you.. so way to fast.
and they dont even get a ticket for speeding!!! :+1:

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 31.12.2015 20:14:11 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197/#comment-24414315

<hr>

Fixed on latest rev.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Dikkedeur
**Original Date:** 31.12.2015 20:15:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197/#comment-24414320

<hr>

i didnt saw something in the commit lord, so i didnt thought it was fixed.
sorry my bad.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 31.12.2015 20:23:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197/#comment-24414360

<hr>

All good. It's because I called update player twice in map update

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 31.12.2015 20:23:56 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197/#comment-24414361

<hr>

Happy new year!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Dikkedeur
**Original Date:** 31.12.2015 20:25:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1197/#comment-24414366

<hr>

you too m8 and let 2016 be the year for great things for oregon!
